device-types/Fortinet/FG-400F.yaml

```diff
@@ -0,0 +1,95 @@
+---
+manufacturer: Fortinet
+model: FortiGate 400F
+slug: fortinet-fg-400f
+part_number: FG-400F
+u_height: 1
+is_full_depth: false
+weight: 6.4
+weight_unit: kg
+airflow: front-to-rear
+front_image: true
+rear_image: true
+comments: '[FortiGate 400F Series Datasheet](https://www.fortinet.com/content/dam/fortinet/assets/data-sheets/fortigate-400f-series.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 214
+    allocated_draw: 109
+  - name: PS2
+    type: iec-60320-c14
+    maximum_draw: 214
+    allocated_draw: 109
```

Comment:
These PSUs are modular. Thus you should create `module-bays` on this device, and then add the associated PSUs to the module-types folder if they do not already exist